packages/core/src/config-engine/config-engine.ts

```diff
@@ -540,6 +540,13 @@ export class DmnoWorkspace {
     }
     throw new Error(`unable to find service - ${descriptor}`);
   }
+  getServiceMetaForBuild(serviceName: string) {
```

Comment:
we'll probably want to include more metadata here, like which plugins are required, what their dependency versions are, which config items in the related services are actually being used, etc